README.md

```diff
@@ -30,7 +30,9 @@ Before you attempt to run this script **from a root shell on the target host** y
 1. have copied the `dkim.private` and `dkim.public` keys generated for the **DKIM** DNS records next to the `remote-install.sh`
 1. make sure you have a large enough scrollback buffer for your terminal to not lose any output
 ## HowTo
-Just ssh as root (passwordless please) to the target host and run the `remote-install.sh` script. Then closely scrutinize the output for any errors.
+The LetsEncrypt configuration is quite OVH specific. If you are running in OVH you can run the install command as `SETUP_LETSENCRYPTOVH=true remote-install.sh` - otherwise you are on your own with the configuration.
```

Comment:
You can run it anywhere, the OVH specific bits have to do with OVH domain hosting-specific steps to confirm domain ownership.

In other words this will work if you host your domain with OVH. If not, you need to configure automatic letsencrypt renewal yourself, using the tools made available by your own domain registrar.

Reply:
Please give me a sign when you want me to merge and the type of merge you would prefer...